app/src/main/java/com/pokedex/koto/pokedex/Pokemon.java

```diff
@@ -4,7 +4,16 @@
  * Created by KOTO on 11/07/2017.
  */
 
-public class responseService {
+public class Pokemon {
```

Comment:
Renamed class, reordered members and added private modifiers (always recommended when no greater visibility is required)